examples/tox21/train_tox21.py

```diff
@@ -51,6 +52,10 @@ def main():
                         default='', help='target label for logistic '
                         'regression. Use all labels if this option '
                         'is not specified.')
+    parser.add_argument('--iterator_type', type=str, choices=iterator_type,
```

Comment:
It is a matter of tastes but I prefer to use hyphens instead of underscores (i.e. `--iterator-type`) because it is visually aligned. `argparse` automatically converts intermediate hyphens into underscores when using this argument as a property. So we can use `args.iterator_type`.